src/match/validators.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_times_list(value):
+    updated_list = []
+    for value_element in value:
+        try:
+            nested_list = value_element.strip().split(",")
+            for index in range(len(nested_list)):
+                if nested_list[index] != "":
+                    nested_list[index] = float(nested_list[index])
+            updated_list.append(str(nested_list)[1:-1])
+        except (ValueError, TypeError):
+            raise ValidationError(
+                _(f"{value} is not a valid list of times"),
+                params={"value": value},
+            )
+    value = updated_list
```

Comment:
does it matter to even set value equal to the updated_list? if you needed to set the original value to updated_list you should've returned here because it looks like value is a string and this equals doesn't update the original value or help with validation at all (you're done when you get here)